app/screens/InfoScreen/SponsorCard.tsx

```diff
@@ -98,20 +93,23 @@ export const SponsorCard = ({
               <Pressable
                 key={index}
                 style={$silverTierButton}
-                onPress={() => openLinkInBrowser(externalURL)}
+                onPress={() => !!externalURL && openLinkInBrowser(externalURL)}
               >
                 <AutoImage
                   {...maxImageDimensions(tier)}
                   accessibilityLabel={sponsor}
                   source={{ uri }}
                 />
-                <Icon icon="arrow" containerStyle={$iconButton} />
+                {externalURL ? <Icon icon="arrow" containerStyle={$iconButton} /> : null}
```

Comment:
We, @joshuayoes & I, thought about moving this out into a wrapper component that was say `MaybeButton`, but for now keeping it inlined.